pkg/globalnet/controllers/ipam/globalnet.go

```diff
@@ -66,6 +66,50 @@ func (i *Controller) initIPTableChains() error {
 		klog.Errorf("error inserting iptables rule %q: %v\n", strings.Join(forwardToSubGlobalNetChain, " "), err)
 	}
 
+	klog.V(log.DEBUG).Infof("Install/ensure %s chain exists", constants.SmGlobalnetEgressChainForPods)
```

Comment:
Won't it be better to combine log msg with the util method? If we don't want to always log, maybe something like `CreateChainIfNotExistsWithLog` ?

Reply:
These logs are printed only once when the Globalnet Pod comes up and will help us while debugging any issues.